client.go

```diff
@@ -107,24 +107,23 @@ func newClient(config Config) (*Client, error) {
 	if name == "" {
 		name = fmt.Sprintf("hz.client_%d", id)
 	}
-	logLevel, err := ilogger.GetLogLevel(config.Logger.Level)
+	serializationService, err := serialization.NewService(&config.Serialization)
 	if err != nil {
 		return nil, err
 	}
-	clientLogger := ilogger.NewWithLevel(logLevel)
-	serializationService, err := serialization.NewService(&config.Serialization)
+	clientLogger, err := loggerFromConf(config)
```

Comment:
Is there a reason why you've changed the creation order of serialization service and logger?

Reply:
I wanted logger to be closer to its first usage `clientLogger.Trace(func() string { return fmt.Sprintf("creating new client: %s", name) })`